Add number of build jobs option to the script (#116)
ricardoasmarques/ceph-dev-docker: README.md

    (docker)# setup-ceph.sh

> If you need to limit the number of build jobs please do this the following
> way for example:

Reviewer: sorry for not noticing this before, but I think the phrase needs some improvement:
```suggestion
If you need to limit the number of build jobs you can do the following:
```

>     NPROC=2 /shared/bin/setup-ceph.sh

Reviewer: ```suggestion
    NPROC=2 setup-ceph.sh
```
